src/rooms/UserAdminRoom.ts

```diff
@@ -49,7 +49,7 @@ export class UserAdminRoom {
         }
         const args: string[] = ev.content.body.split(" ");
         const command = args[0].toLowerCase();
-        log.info(`${this.userId} sent admin message ${args[0].substr(32)}`);
+        log.info(`${this.userId} sent admin message ${args[0].substring(32)}`);
```

Comment:
Why would we skip the first 32 characters of a admin message?
If this is trying to truncate the command to a maximum of 32 characters:
```suggestion
        log.info(`${this.userId} sent admin message ${args[0].length > 32 ? `$args[0].substring(0, 32)}…` : args[0]}`);
```

Reply:
Hah, probably an oversight – unless we're trying to hide credentials in some quirky way.

The ternary is not needed, substr() won't "lenghten" the string if it's shorter:
```
> "foo".substr(0, 32)
'foo'
```

Reply:
I added the ternary to append the ellipses so you know that some characters at the end are missi…

Reply:
@Half-Shot You've last touched this code.
Do you remember if skipping the first 32 characters is preferred over limiting the length?

Reply:
> I added the ternary to append the ellipses so you know that some characters at the end are missi…

Ah, indeed, didn't notice that. It might be an off-by-one error if you have 33 characters, but it's no big deal :)

Reply:
I can't remember this one, but it sounds like that's what I was trying to achieve.